lmfdb/lfunctions/main.py

```diff
@@ -349,14 +349,21 @@ def common_parse(info, query):
                  lambda origins: " ".join('<a href="%s">%s</a>' % (url, name) for name, url in origins),
                  default=True)],
     db_cols=['algebraic', 'analytic_conductor', 'bad_primes', 'central_character', 'conductor', 'degree', 'instance_urls', 'label', 'motivic_weight', 'mu_real', 'mu_imag', 'nu_real_doubled', 'nu_imag', 'order_of_vanishing', 'primitive', 'rational', 'root_analytic_conductor', 'root_angle', 'self_dual', 'z1'])
-lfunc_columns.dummy_download = True
+# lfunc_columns.dummy_download = True
```

Comment:
This looks like an unrelated change that accidentally got included in this PR (certainly you should change the name of the class from `G2C_download` to `Lfunc_download` eventually).

Reply:
yes indeed, this was from another thing ... removed now